packages/init/package.json

```diff
@@ -14,5 +14,14 @@
     "chalk": "^2.4.1",
     "jscodeshift": "^0.5.0",
     "p-each-series": "^1.0.0"
+  },
+  "devDependencies": {
+    "@types/node": "^10.3.3",
+    "@types/p-each-series": "^1.0.0",
+    "typescript": "^2.9.2"
+  },
+  "scripts": {
+    "build": "tsc",
```

Comment:
This should go in webpack-cli `package.json`

Reply:
Updated.

Reply:
Actually, this package is not recognizing `@types/node` if I run `tsc` from root. Still trying to trace what is the issue.

Reply:
For temp. testing purpose please run `tsc` or `npx tsc` from `packages/init` path.

Reply:
No temp fixes on this :) Resolve the issue rather than creating hacky solutions that might break later